databricks/koalas/tests/test_dataframe.py

```diff
@@ -2070,28 +2070,44 @@ def test_drop_duplicates(self):
         )
         kdf = ks.from_pandas(pdf)
 
-        # inplace is False
-        self.assert_eq(pdf.drop_duplicates().sort_index(), kdf.drop_duplicates().sort_index())
-        self.assert_eq(pdf.drop_duplicates("a").sort_index(), kdf.drop_duplicates("a").sort_index())
-        self.assert_eq(
-            pdf.drop_duplicates(["a", "b"]).sort_index(),
-            kdf.drop_duplicates(["a", "b"]).sort_index(),
-        )
+        for keep in ["first", "last", False]:
+            with self.subTest():
```

Comment:
Shall we add `keep=keep` to be able to see the difference?
Otherwise all the error titles are something like:
```
ERROR: test_drop_duplicates (databricks.koalas.tests.test_dataframe.DataFrameTest) (<subtest>)
```

instead of:

```
ERROR: test_drop_duplicates (databricks.koalas.tests.test_dataframe.DataFrameTest) (keep='first')
```

and so on.

Reply:
Done!